src/main/java/com/minecolonies/coremod/blocks/BlockInfoPoster.java

```diff
@@ -58,11 +57,13 @@ private void initBlock()
     }
 
     @Override
-    public TileEntity createNewTileEntity(final World worldIn, final int meta)
+    public TileEntity createNewTileEntity(@NotNull final World worldIn, final int meta)
     {
         return new TileEntityInfoPoster();
     }
 
+    @NotNull
+    @Deprecated
     @Override
     public IBlockState getStateFromMeta(final int meta)
     {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1133)